packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,651 @@
+
+### About Input
+
+Use an input when you want to allow a user to enter a single line of text. By default, text inputs allow users to enter any type of text (letters, numbers, and symbols).
+You can set the input to more [specific types](#input-types) if needed.
+
+### Accessibility
+
+<UnorderedList>
+  <ListItem>Include a visible label on <strong>all</strong> form fields.</ListItem>
+  <ListItem>
+    Don't use placeholder text as a replacement for labels. It can be used to provide an example to users, but will
+    disappear from the field when a user enters text. It's also not broadly supported by assistive technologies and
+    won't display in older browsers.
+  </ListItem>
+  <ListItem>
+    Use one of 3 ways to label a form field:
+    <UnorderedList marginBottom="space0">
+      <ListItem>Visible label with <Anchor href="#formlabel-props">FormLabel</Anchor> (preferred)</ListItem>
+      <ListItem>Visible label that's associated to the input with <inlineCode>aria-labelledby</inlineCode></ListItem>
+      <ListItem>Label directly using <inlineCode>aria-label</inlineCode></ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Provide clear identification of required fields in the label or at the start of a form. If you use the required
+    field indicator, include the required field key at the start of the form.
+    <UnorderedList marginBottom="space0">
+      <ListItem>Use the <inlineCode>required</inlineCode> prop to programatically indicate they are required to browsers.</ListItem>
+      <ListItem>
+        The aria-required attribute informs assistive technologies about required controls so that they are
+        appropriately announced to users. This is only required on form fields that don't support the required attribute
+        from the HTML spec.
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Include inline error text with the error icon on any field that errors to make it visually clear that the field
+    changed.
+  </ListItem>
+</UnorderedList>
+
+## Examples
+
+### Input
+
+The input component should include the base input, along with supporting elements to compose an input field that gives users the context they need to successfully fill it out.
+
+- **Label** — A label should be included for every form field and provide a short title for its associated input.
+- **Required field indicator** — In a form where there are fewer or as many required fields as optional, use a required indicator to show users which fields are required to be filled out.
+- **Help text** — Text that's placed below the field to help users prevent an error and describe what makes the form field successful.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="email_address" required>Email Address</FormLabel>
+  <FormInput id="email_address" name="email_address" type="email" placeholder="example@twilio.com" onChange="" required/>
```

Comment:
Anywhere there is supplimentary text to the input field, help or error, you should describe the input by the text. aria-describedby is a IDREF field, so it takes ids to the elements you are describing by.

```suggestion
  <FormInput aria-describedby="email_help_text" id="email_address" name="email_address" type="email" placeholder="example@twilio.com" onChange="" required/>
```

Reply:
We should update in all cases in the doc and probably call out this requirement in the accessibility section and as a `CallOut` about this first example and the first error example,

Reply:
Updated an added a `Callout`.